### DISCUSSION TITLE
Buffers not showing after sourcing session

### DISCUSSION THREAD

**Discussion by gmnz** [nvim-neo-tree/neo-tree.nvim#1138] [Q&A] [answered]
Hey,

When I source a session file, bufferline and telescope show all my buffers from the session, but neo tree show only the first one that was focused after sourcing session (using command "Neotree buffers toggle"). Also, when focusing the other buffers, neo tree adds some of them but not all. I have to toggle it to make all the buffers I had focused in the meanwhile appear. Does anybody else experience this behaviour? Can I somehow make it behave like bufferline (updates buffer list without toggling, showing all buffers right after source)?

**Comment by cseickel** [comment, marked as answer]
Explanation and fix here: https://github.com/nvim-neo-tree/neo-tree.nvim/issues/1124#issuecomment-1695965454

**Comment by gmnz** [comment]
Thanks, works great! I actually saw that one in the readme, but the word "unloaded" didn't sound like the thing I wanted and in the example in readme it was set to true, so I thought that was the default value anyhow, so that can't be the problem. Sorry, I could have dug deeper.
